### DISCUSSION TITLE
Option to show time of last ping instead of time duration since last ping

### DISCUSSION THREAD

**Discussion by StephenWetzel** [healthchecks/healthchecks#1029] [Ideas]
On the main list of checks, in the "Last Ping" column, right now they are all relative times, like "12 hours ago".  I would also like to be able to see the absolute time the ping happened, like "2024-07-18 04:13:34".  I know I can click into it to see the times, but I'd like something easier.  Ideally, there would be a setting to allow me to toggle between them, but if you don't want add that, just showing the time when you hover over the relative time seems to be a pretty standard UI.  See for [example the Github UI itself](https://github.com/healthchecks/healthchecks/issues) where if you hover over time like "last week" it shows the timestamp.

**Comment by cuu508** [comment]
We could add a timestamp in a tooltip, similar to GitHub.

I'm not sure what timezone to use though:

* UTC?
* Browser's timezone?
* If it's a cron or systemd check, the timezone configured for the check?
* The timezone specified in Account Settings > Email Reports?

I think this was also why I originally opted for the relative "12 hours ago" format, as the timezone doesn't matter then.

In the Ping Details dialog we get around the issue [by showing the date in multiple formats](https://github.com/healthchecks/healthchecks/issues/975#issuecomment-2137327052) but this would not look good in a tooltip.

**Reply by StephenWetzel** [reply]
My vote would be for browser's timezone, although really any of those choices make sense to me.  Personally, I like when times update to where I am now, using the browser's timezone, as it seems unlikely that if I were traveling I would want to know what time something ran in the timezone I'm based in normally vs what time it actually is where I am now.

UTC is always a safe default too, but I am always just translating UTC times into my local time, so my preference is to just display local time.
